python/cuxfilter/charts/core/non_aggregate/core_stacked_line.py

```diff
@@ -21,7 +21,6 @@ class BaseStackedLine(BaseChart):
     y_range: Tuple = None
     use_data_tiles = False
     y: list = []
-    colors: list = []
```

Comment:
This one doesn't look right to me. I don't see `self.colors` getting set in `__init__()`.

https://github.com/rapidsai/cuxfilter/blob/53537fe4fb2dff70789d8cfb81d2c058adc6488c/python/cuxfilter/charts/core/non_aggregate/core_stacked_line.py#L59

And I found at least one place that appears to rely on this attribute being defined on an instance:

https://github.com/rapidsai/cuxfilter/blob/53537fe4fb2dff70789d8cfb81d2c058adc6488c/python/cuxfilter/charts/datashader/plots.py#L384

https://github.com/rapidsai/cuxfilter/blob/53537fe4fb2dff70789d8cfb81d2c058adc6488c/python/cuxfilter/charts/datashader/plots.py#L394

What was the the `flake8` error that this change addresses? Maybe there's another way we could address it, like a more specific type hint in the initialization of `colors`.

Reply:
It's defined as a property here: https://github.com/rapidsai/cuxfilter/blob/f9a617d889bd01d45687e16f96beebc06f6acdfd/python/cuxfilter/charts/core/non_aggregate/core_stacked_line.py#L52-L56

flake8 was mad that this object had both an attribute and a property with the same name, and it seemed to me that the property was the correct choice to keep. What do you think?

Reply:
ahhhh my `git grep -E '\.colors`' wouldn't have caught it being defined with `@property`, should have thought of that!

Ok then yeah this is totally fine, and I agree with preferring the `@property` one. Thanks for explaining that.

Reply:
cc: @AjayThorve for awareness

Reply:
thanks, this seems correct, and yes property is the right one to keep